AK/Format.h

```diff
@@ -717,13 +717,11 @@ struct Formatter<ErrorOr<T, ErrorType>> : Formatter<FormatString> {
 
 template<typename T>
 struct Formatter<Optional<T>> : Formatter<FormatString> {
-    static constexpr bool is_debug_only() { return true; }
-
     ErrorOr<void> format(FormatBuilder& builder, Optional<T> const& optional)
     {
         if (optional.has_value())
-            return Formatter<FormatString>::format(builder, "Optional({})"sv, *optional);
-        return builder.put_literal("OptionalNone"sv);
+            return Formatter<FormatString>::format(builder, "{}"sv, *optional);
+        return builder.put_literal("None"sv);
```

Comment:
itʼd be nice to differentiate “None” content from empty state somehow. maybe a symbol before? or quote non-empty?

Reply:
This is how jakt has it currently, can't say I remember such an ambiguity showing up